torch/nn/utils/fusion.py

```diff
@@ -12,16 +12,16 @@ def fuse_conv_bn_eval(conv, bn):
     b_conv = fused_conv.bias
 
     bn_mean = bn.running_mean
-    bn_var_sqrt = torch.sqrt(bn.running_var + bn.eps)
+    bn_var_sqrt = torch.rsqrt(bn.running_var + bn.eps)
```

Comment:
@colesbury once told me that `rsqrt` isn't really better than division with `sqrt`.

Reply:
@colesbury Sam, can you comment on that, please?

Reply:
torch.rsqrt just computes `1.0 / sqrt(x)`. It's not a fast approximation (unlike x86 rsqrtss instruction or rsqrt in CUDA). At one point it wasn't even fused on the CPU (it is now).

You do lose a little precision this way because there are now three rounded operations:

```
x * (1.0 / sqrt(y))
```

instead of just two:

```
x / sqrt(y)
```

I'm a little confused about the intention behind this change. Is it any faster? If so, does that matter? (Isn't fusing conv + batch norm a one-time operation?)

Anyways, it isn't **bad** to use torch.rsqrt, just don't necessarily expect any performance gains.

Reply:
The only motivation behind this change was old memory of Patterson & Hennesy's book saying division is much more expensive than multiplication 😄 However, if we lose precision in that case -- I would rather sacrifice a little perf but not lose precision